CMakeLists.txt

```diff
@@ -1,123 +0,0 @@
-cmake_minimum_required(VERSION 3.0)
```

Comment:
Even though I lean to think that cmake is useless compared to meson :), I think you may want to keep this file :)

Reply:
file restored